test/utils/operator/orderStatus.test.ts

```diff
@@ -114,6 +114,9 @@ describe('Filled status', () => {
 })
 
 describe('Canceled status', () => {
+  const currentDate = new Date(DATE.setFullYear(DATE.getFullYear() + 4))
```

Comment:
clarify please what's that 4 magic number about

Reply:
Done!